fcrepo/fcrepo
# Enable configurable Pairtree pid minter

**Enable configurable pair tree pid minter.**
* * *

**JIRA Ticket**: https://fedora-repository.atlassian.net/jira/software/c/projects/FCREPO/issues/FCREPO-3916

# What does this Pull Request do?
At the request of the Samvera Fedora 6 Working group, I have enabled pair tree style PID minting in Fedora 6

# How should this be tested?

A description of what steps someone could take to:
* set properties ```fcrepo.pid.minter.length```  (2) and ```firepower.pid.minter.count``` (4) to replicate the Fedora PairTree PIDs of old.  Easiest way is ``` mvn jetty:run -Dfcrepo.pid.minter.length=2 -Dfcrepo.pid.minter.count=4 ```
* Try creating new containers in the UI. PIDs should resemble PairTree IDs 
* If you don't set the properties, standard UUID PIDs will be generated


# Interested parties
Tag (@ mention) interested parties or, if unsure, @fcrepo/committers
